packages/manager/apps/web/client/app/hosting/database/dump/hosting-database-dump.controller.js

```diff
@@ -14,8 +12,8 @@ export default class DatabaseDumpsCtrl {
     $translate,
     $window,
     Alerter,
-    HostingDatabase,
     databaseLink,
+    HostingDatabase,
```

Comment:
Why you change the sort to constructor inject list. Because if you want do a sort the logical sort would be an alphabetical sort.

Reply:
I will do some logical alphabetical lexicographical sort then ;)